crates/turborepo-auth/src/auth_file.rs

```diff
@@ -0,0 +1,192 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use turbopath::AbsoluteSystemPath;
+use turborepo_api_client::Client;
+
+use crate::Error;
+
+#[derive(Serialize, Deserialize, Debug, Default)]
+/// AuthFile contains a list of domains, each with a token.
+pub struct AuthFile {
+    pub tokens: HashMap<String, String>,
```

Comment:
I can add a function to get them like `pub fn tokens()` and `pub fn tokens_mut`, but maybe it's worth discussing how much functionality to put on the `AuthFile` vs the hashmap of tokens itself. Mostly because logout will need to mutate the tokens, which in the current state just calls the hashmap remove, but I can see an argument for the authfile to have a remove function as well.